### ISSUE TITLE
Button focus state not super clear in dark mode

### ISSUE THREAD

**Issue by BNewing** [alphagov/govuk-design-system#1059]
I've been using the 'Dark Mode' Chrome plugin for a workshop around accessibility testing lately. 

One thing we've noticed is that previously, the Design System button had a clear focus state with the dark mode turned on: 

<img width="251" alt="image" src="https://user-images.githubusercontent.com/16868713/66475164-b421a100-ea8a-11e9-9c6b-08eada04b018.png">

However at present, when focused, it looks like this for me: 

<img width="233" alt="image" src="https://user-images.githubusercontent.com/16868713/66475197-c00d6300-ea8a-11e9-9be3-4f8ba059aae2.png">

Focus is indicated by the black border on the bottom, but it's a bit hard to spot against the background.

**Comment by nickcolley** [comment]
Thanks for raising this!

We aim to support [users who adjust or override the colours of websites they visit][how-users-change-colours-on-websites]. We test this by [changing colours in Firefox][changing-colours-in-firefox], by [enabling 'High Contrast' mode in Windows][enabling-high-contrast-mode-in-windows] and by using the [High Contrast plugin for Chrome][high-contrast-plugin-for-chrome].

[service-manual-assistive-technologies]: https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies#what-to-test

[changing-colours-in-firefox]:
https://support.mozilla.org/en-US/kb/change-fonts-and-colors-websites-use

[enabling-high-contrast-mode-in-windows]:
https://support.microsoft.com/en-gb/help/13862/windows-use-high-contrast-mode

[high-contrast-plugin-for-chrome]: https://chrome.google.com/webstore/detail/high-contrast/djcfdncoelnlbldjfhinnjlhdjlikmph?hl=en-US

[how-users-change-colours-on-websites]:
https://accessibility.blog.gov.uk/2017/03/27/how-users-change-colours-on-websites/

I've managed to reproduce this with https://chrome.google.com/webstore/detail/dark-theme-for-chrome/geooakdjiamlhpechokegobmhdmlgidk?hl=en

Dark mode functionality is becoming more stable as a real feature and there is a backlog conversation where this is being discussed: https://github.com/alphagov/govuk-design-system-backlog/issues/177

I think in the future we will want to review the standards based approach to dark modes cross browser and consider improvements based on that, but making changes for Chrome plugins outside of standards will not be sustainable.

I'm going to close this based on that but feel free to continue the conversation on the backlog as we track the standards for dark modes... :)

**Comment by BNewing** [comment]
Ah! Sorry, I should have specified the plugin I was using. I've been using one called Dark Mode, which has both a [Firefox](https://addons.mozilla.org/en-GB/firefox/addon/dark-mode-webextension/) and [Chrome](https://chrome.google.com/webstore/detail/dark-mode/dmghijelimhndkbmpgbldicpogfkceaj?hl=en) version. I'd chosen it because it was available in both.

I will have a think about switching for the workshops I run :) very interested in ongoing discussions on this!
